lib/mix/tasks/phoenix_slime.gen.html.ex

```diff
@@ -0,0 +1,147 @@
+defmodule Mix.Tasks.PhoenixSlime.Gen.Html do
+  use Mix.Task
+
+  @shortdoc "Generates controller, model and views for an HTML based resource"
```

Comment:
I don't like the idea that it's a slime resource because that's only the template. The controller, model, and view are all agnostic. The route is likely HTML specific.